Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -210,7 +221,9 @@ def process_incident_create_message(demisto_user: dict, message: str) -> str:
             incidents: Union[dict, list] = json.loads(incidents_json.replace('“', '"').replace('”', '"'))
             if not isinstance(incidents, list):
                 incidents = [incidents]
-            created_incident = create_incidents(demisto_user, incidents)
+
+            add_req_data_to_incidents(incidents, request_body)  # type: ignore[arg-type]
```

Comment:
i don't recommend sending the whole request_body as it is, it includes confidential data

Reply:
Good to know, let's sync offline on which values are confidential.